partner-portal/src/UI/src/app/features/auth/enums/call-back.enum.ts

```diff
@@ -0,0 +1,6 @@
+
+export const CallStatusDescription = new Map<number | undefined, string>([
+  [0, 'Open'],
+  [1, 'Closed']
+])
```

Comment:
move to a different folder. I'm not really sure what folder, there is a folder structure I try to follow that other developers follow, I'll see if I can find the link

Reply:
https://itnext.io/choosing-a-highly-scalable-folder-structure-in-angular-d987de65ec7

And after skimming through it I still don't know where to put it

Reply:
/shared/enums

https://github.com/bcgov/moh-pidp/blob/develop/workspace/apps/pidp/src/app/shared/enums/roles.enum.ts

Reply:
@jburditt  I have added this folder to shared enums level where the enums for the identity provider exists